README.md

```diff
@@ -18,6 +18,12 @@ The samples requires the below requirements to run.
 * [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
 * .Net Framework 4.5 and above
 
+In order to download samples in the Report Writer Properly , run the below mwntioned command.
```

Comment:
Can we change to the following?

To export the data visualization report items in report writer, run the below mentioned command. Without this, the data visualization report items no longer available in the report exported documents only for the report writer.

Reply:
Updated